ci/test_wheel.sh

```diff
@@ -3,10 +3,10 @@
 
 set -euo pipefail
 
-package_name="ucx"
+package_name="libucx"
 
 WHEELHOUSE="${PWD}/dist/"
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
-RAPIDS_PY_WHEEL_NAME="${package_name}_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp "${WHEELHOUSE}"
+RAPIDS_PY_WHEEL_NAME="${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp "${WHEELHOUSE}"
```

Comment:
Is the removal of `${package_name}_` intentional? Won't that then make the wheel name just the CUDA suffice?

Reply:
Yes, this is intentional. The tl;dr is that we're reworking CI scripts so that instead of creating a separate tarball for each wheel, we'll just have two jobs per repo, one that builds the C++ wheel and one that builds all the Python wheels, so we only need to delineate C++ vs Python and CUDA version. There's going to be a bit of an awkward transition here, but this line is fine for now.